homeassistant/components/intellifire/strings.json

```diff
@@ -1,12 +1,16 @@
 {
   "config": {
+    "flow_title": "Discovered Fireplace ({host})",
     "step": {
       "manual_device_entry": {
         "description": "Local Configuration",
         "data": {
           "host": "[%key:common::config_flow::data::host%]"
         }
       },
+      "dhcp_confirm": {
+        "description": "Setup {host}"
```

Comment:
Maybe change this to a question?  'Do you want to set up {host}?'

Reply:
Happy to do so -> I was thinking about your earlier "Manual IP address Selection" => "IP ADDRESS" because it was easier to translate... 

I'd much prefer to do something like this ... also I need to do a local test on the last changes to verify stuff

Reply:
I updated the description with the new image... but it looks like stuff is working on my end - resolving this one.